src/components/MarkdownOutput/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import { ReactNode } from 'react';
+import { BaseProps } from '../types';
+
+export interface MarkdownOutputProps extends BaseProps {
+    id?: string;
+    renderers?: object;
+    plugins?: object[];
+    parserOptions?: object;
+    source: string;
```

Comment:
- source => value
- remove renderers , plugins, parserOptions(we wont allow that for now) 
- source is not required(by default empty string)